vortex-dtype/src/ptype.rs

```diff
@@ -88,6 +100,22 @@ macro_rules! native_ptype {
                 self.cmp(&other)
             }
 
+            fn maybe_null_add(self, other: Self) -> Self {
```

Comment:
I don't think we should define these operators, and they _almost certainly_ should not be wrapping